### DISCUSSION TITLE
Separate Session for Separate Instances dont work.

### DISCUSSION THREAD

**Discussion by kreationnext** [cefsharp/CefSharp#3420] [General]
Separate Sessions for different instances was working well in version 79. But after that version, the cache path doesn't work this way.

                RequestContextSettings rcs = new RequestContextSettings();
                string sourceDir = System.IO.Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), StringStorage.PROCESS_CACHE);
                rcs.CachePath = System.IO.Path.Combine(sourceDir, "1");
                rcs.IgnoreCertificateErrors = true;
                rcs.PersistSessionCookies = true;
                rcs.PersistUserPreferences = true;

                google_browser = new ChromiumWebBrowser();
                google_browser.RequestContext = new RequestContext(rcs);
                google_browser.IsBrowserInitializedChanged += GoogleBrowser_IsBrowserInitializedChanged;
                var x = google_browser.RequestContext.CachePath;
x => its is always blank.

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/issues/3111#issuecomment-629713608
